.gitignore

```diff
@@ -36,4 +36,4 @@ ehthumbs.db
 *.sublime-project
 *.sublime-workspace
 
-
+Icons.meta
```

Comment:
I ended up taking this out because I worried it might have had something to do with our disappearing references. I'm willing to give it a try again though and find out!

Reply:
Ohhh I see. Ok!